docs/static/templates/document-planning-template.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Doc planning template
+menuTitle: Doc Planning
+description: Use this template when you write a documentation plan.
+aliases:
+  - /docs/writers-toolkit/latest/templates/document-planning-template
+weight: 100
+keywords:
+  - doc planning
+  - documentation plan
+  - writing new documents
+---
+<!-- Refer to [Topic front matter]({{< relref "../../front-matter/" >}}) for more information about how to populate front matter. -->
+
+# Documentation plan
+
+A typical documentation plan consists in an introduction, proposal, and content outline.
```

Comment:
If we're going to publish this publicly as part of the Writer's Toolkit, it probably needs some explanation about what level of documentation might require a documentation plan.  We don't want to unduly burden our contributors.

Reply:
I agree.

Reply:
This file feels like a bit of scope creep... there are changes about how to write documentation, and this discusses how to plan your documentation. It think it warrants a subsequent discussion with a shift in focus.